modules/javafx.graphics/src/main/resources/com/sun/prism/es2/glsl/diffuse_color.frag

```diff
@@ -27,7 +27,10 @@
 
 #ifdef GL_ES
 
+#ifndef EXTENSION_APPLIED
 #extension GL_OES_standard_derivatives : enable
+#define EXTENSION_APPLIED
```

Comment:
Usual practice is to put the `#define` right after the `#ifndef`, but I don't have a strong opinion about that.

Reply:
Makes sense, I've applied the change